tools/autograph-monitor/contentsignature.go

```diff
@@ -21,9 +21,9 @@ import (
 // we don't want to alert on those.
 // https://bugzilla.mozilla.org/show_bug.cgi?id=1466523
 var ignoredCerts = map[string]bool{
-	"fingerprinting-defenses.content-signature.mozilla.org": true,
-	"fennec-dlc.content-signature.mozilla.org":              true,
-	"focus-experiments.content-signature.mozilla.org":       true,
+	// "fingerprinting-defenses.content-signature.mozilla.org": true,
+	// "fennec-dlc.content-signature.mozilla.org":              true,
+	// "focus-experiments.content-signature.mozilla.org":       true,
```

Comment:
was this for testing?

Reply:
it actually needs to be deleted , good catch!

Reply:
Ah, no, I actually wanted to keep this here to remember the syntax, but it should be commented out because these certs are now gone.